src/VisualStudio/CSharp/Impl/Options/IntelliSenseOptionPageControl.xaml.cs

```diff
@@ -39,5 +43,19 @@ private void BringUpOnIdentifier_Unchecked(object sender, System.Windows.RoutedE
             ShowKeywords.IsEnabled = true;
             ShowSnippets.IsEnabled = true;
         }
+
+        private void Show_completion_list_after_a_character_is_deleted_Checked(object sender, RoutedEventArgs e)
```

Comment:
I don't think you need these methods - `BindToOption` should handle it. We only have these for options whose values are tied to other options.